samples/vault/README.md

```diff
@@ -3,7 +3,7 @@ This sample shows how to port an existing Go application to EGo.
 
 First clone the repo:
 ```sh
-git clone --depth=1 --branch=v1.6.3 https://github.com/hashicorp/vault
+git clone --depth=1 --branch=release/1.6.x https://github.com/hashicorp/vault
```

Comment:
I think it's better to pin this to the tag like before instead of to the branch which will change over time.